products/products.json

```diff
@@ -0,0 +1,397 @@
+[
+    {
+        "id": 1,
+        "category": "phones",
+        "brand": "Apple",
+        "model": "I Phone X",
+        "price": 60000,
+        "discount": 0,
+        "description": "",
+        "images": ["./i-phone-x/iphonex-1.jpg", "./i-phone-x/iphonex-2.jpg", "./i-phone-x/iphonex-3.jpg", "./i-phone-x/iphonex-4.jpg"],
+        "details": {
+            "camera": "32 Mp",
+            "color": "white",
+            "RAM": "8GB",
+            "ROM": "64GB"
+        },
+        "payment": {
+            "payments": 6,
+            "price-payment": 10000,
+            "free-interest": true
+        }
+    },
+    {
+        "id": 2,
+        "category": "phones",
+        "brand": "Motorola",
+        "model": "G7",
+        "price": 15000,
+        "discount": 10,
+        "description": "",
+        "images": ["./motorola/motorola-g7-1.jpg", "./motorola/motorola-g7-2.jpg", "./motorola/motorola-g7-3.jpg", "./motorola/motorola-g7-4.jpg"],
+        "details": {
+            "camera": "8 Mp",
+            "color": "black",
+            "RAM": "2GB",
+            "ROM": "16GB"
+        },
+        "payment": {
+            "payments": 3,
+            "price-payment": 5000,
```

Comment:
This is not a good way to create a property. Remember we can access to a property using `ObjectName.property` or `ObjectName['property']` but if you use "-" in property name, you can't call to a property using the second one way (`payment.price-payment` is not valid, you can't use "-" in a variable name)